pkg/R/utils.R

```diff
@@ -164,11 +176,11 @@ reserialize <- function(rdd) {
   if (!inherits(rdd, "RDD")) {
     stop("Argument 'rdd' is not an RDD type.")
   }
-  if (rdd@env$serialized) {
-    return(rdd)
-  } else {
+  if (rdd@env$serialized == "string") {
```

Comment:
I lean to agree with @shivaram. Even though current version of reserialize does not actually convert the `row` type, it might be useful to include Row if it does not break other tests.